pkg/analysis_server/lib/src/computer/computer_closingLabels.dart

```diff
@@ -7,13 +7,22 @@ import 'package:analyzer/dart/ast/ast.dart';
 import 'package:analyzer/dart/ast/visitor.dart';
 import 'package:analyzer/src/generated/source.dart';
 
+class _ClosingLabelWithLineCount {
+  final ClosingLabel label;
+  final int spannedLines;
+
+  _ClosingLabelWithLineCount(ClosingLabel label, int spannedLines)
+      : this.label = label,
+        this.spannedLines = spannedLines;
+}
```

Comment:
You could write the (whole) constructor as
```
_ClosingLabelWithLineCount(this.label, this.spannedLines);
```
This is semantically the same as what you wrote, but a little more compact.